api/v1beta1/volumereference.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2021 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1beta1
+
+// VolumeReference ...
+type VolumeReference struct {
+	// This must match the Name of a ConfigMap.
+	Name string `json:"name" protobuf:"bytes,1,opt,name=name"`
+	// Path within the container at which the volume should be mounted.  Must
+	// not contain ':'.
+	// +optional
```

Comment:
Isn't `MountPath` mandatory?

Reply:
Just found that default path is applied when not provided. Can you please update the description to mention the default path to be used if not specified?

Reply:
MountPath default value is `operator.KogitoHomeDir + "/config"`. Its not mandatory for user.

Reply:
yes, but it would be good to mention the default path in comment so it become part of CRD. That way users can easily find it.